modules/OCPP/OCPP.cpp

```diff
@@ -414,4 +410,28 @@ void OCPP::ready() {
     invoke_ready(*p_auth_provider);
 }
 
+ocpp1_6::ChargePointErrorCode OCPP::getOcppErrorCode(json evse_error) {
```

Comment:
furthermore, this is a completely static helper function, it doesn't need to be attached to the OCPP object, you could equally just write
static ocpp1_6::ChargePointErrorCode getOcppErrorCode(json evse_error)
also note, that we mostly use snake case for function names, that would be get_ocpp_error_code